src/test/resources/specs/name-matching/Full_stops_in_names.feature

```diff
@@ -73,6 +73,20 @@ Feature: Names with full stops
       | Bb.         | Ddd       | 1987-12-10    | SE123456B |
     And a Matched response will be returned from the service
 
+  @name_matching
+  Scenario: Name when middle name has a full stop and a space and is matched with HMRC 2
```

Comment:
I think the old test is now redundant / superseded by this one. Might have been your intention when you copied it here - sorry for not spotting when I was watching.